### PR TITLE
Nrcan

### PR DESCRIPTION
**Pull request by phylroy**
Merged updates to NECB archetypes.

### CODE REVIEW

**Comment by asparke2** [comment]
@phylroy is this ready to go?

**Comment by phylroy** [comment]
Yep it is! There is a more up to date version I was going to do a pull request on..though.. Should you pull this in...or should I cancel this one and send a more recent one?

**Comment by asparke2** [comment]
The PR is attached to this branch, so as long as all the commits are on this branch they will come on.  If there is a different branch with the more up-to-date changes, you could just merge them into this branch.

**Comment by phylroy** [comment]
Okay the nrcan branch is ready I just merge with master's HEAD this morning, so it is good to go.

File: openstudio-standards/lib/openstudio-standards/prototypes/Prototype.utilities.rb [hunk 1]
```diff
@@ -145,14 +145,8 @@ def hspf_to_cop_heating_no_fan(hspf)
...
 def cop_to_seer(cop)
-  seer = nil
-
-  # First convert COP to EER
-  eer = cop_to_eer(cop)
-
-  # Next convert from EER to SEER
-  delta = 1.1088**2 - 4.0 * 0.0182 * eer
-  seer = (-delta**0.5 + 1.1088) / (2.0 * 0.0182)
+  delta = 0.3796**2 - 4.0 * 0.0076 * cop
```
Comment (asparke2) [inline openstudio-standards/lib/openstudio-standards/prototypes/Prototype.utilities.rb]: Do you have a citation for this change?  Right now, the comments in the method point to this document by name:

https://www.energycodes.gov/sites/default/files/documents/BECP_Energy_Cost_Savings_STD2010_May2011_v00.pdf

File: openstudio-standards/test/test_primary_school.rb [hunk 2]
```diff
@@ -7,8 +7,8 @@ class TestPrimarySchool < CreateDOEPrototypeBuildingTest
 
   building_types = ['PrimarySchool']
   
-  templates = ['DOE Ref Pre-1980','DOE Ref 1980-2004','90.1-2010']
-  climate_zones = ['ASHRAE 169-2006-2A','ASHRAE 169-2006-3B','ASHRAE 169-2006-4A','ASHRAE 169-2006-5A']
+  templates = ['DOE Ref Pre-1980']
```
Comment (asparke2) [inline openstudio-standards/test/test_primary_school.rb]: Were these turned off for runtime or something?  I'd still like to test these combos.

File: openstudio-standards/lib/openstudio-standards/prototypes/Prototype.utilities.rb (hunk 1 above)
Comment (khaddad) [inline openstudio-standards/lib/openstudio-standards/prototypes/Prototype.utilities.rb]: @asparke2, This new method 'cop_to_seer' is based on method 'seer_to_cop_cooling_no_fan'. It uses then the same methodology to get cop from seer. So the reference listed in the existing method still applies in the new method.

**Comment by phylroy** [comment]
Hey @asparke2  and @khaddad , 

I noticed that the nrcan branch is missing after the merge....? We will be using the nrcan branch as our internal 'stable' branch, as some of our script rely on this branch. Also a specialized version of OpenStudio-Server version pulls this in as well. We will however, delete the horribly named 'migrate_necb_spacetypes...' branch. So I will restore the branch. 

Cheers,
P

**Comment by asparke2** [comment]
Ok, sorry about that.  I reflexively delete old branches after they've been merged.

**Pull request merged**